### DISCUSSION TITLE
Week 13 - Final Project Post

### DISCUSSION THREAD

**Discussion by derekscholes** [USU-ITLS-x265-23-summer/class-repo#237] [General]
Overall, I think my project went fairly well. There is a lot I can do to continue to improve on it but compared to what I knew before and what I have been able to accomplish, I would say that I have improved a lot. Here is a little estimate of how I think I did. 

## Breadth Rubric:
**HTML:** Intermediate to Advanced
- I feel like I did an intermediate job in this section due to the number of pages I have created and the content I have published. I am unsure if I would say advanced but due to the quantity of HTML, I might say advanced.

**CSS:** Intermediate
- I also believe I did intermediate in CSS as I was able to use Flexbox, create organized columns and content for my CV and improve the basic formatting of my site, and make it slightly appealing. 

**Java:** Basic
- My Java was very basic. I wasn't too sure how to make many things function or what to add. All I ended up doing was creating a dropdown blog. I attempted a dropdown menu that functioned but I didn't like how it looked on my site. I also wanted to add more but might do that at a later date as I continue to update site content.

**Accessibility:** Basic
- I am unsure how basic my accessibility is. I know color contrast can improve some, but I attempted to make text and buttons large for easy visualization and to organize things so items were easy to find. 

## Final Thoughts:
I believe I did very well on this project based on my previous knowledge (which was none) and my ability to work with HTML, CSS, and Java. I think I met the minimum requirements for this project and hope to continue pursuing this in the future as a way to develop my skills and future business ideas. Let me know what improvements you all think should be added to my site, and thanks again for all the comments I have received. If you have any questions for me, feel free to ask!

[Project Site](https://www.ginutritiontherapy.com)
[GitHub Content](https://github.com/derekscholes/derekscholes.github.io)

**Comment by gabebott** [comment]
Hey Derek, everything looks great from your code on your content page. That link for the project site wasn't loading for me. Let me know if you get that working, I would love to take a look at it!

**Reply by derekscholes** [reply]
My apologies, I have fixed the link and it should work now.

**Comment by audreyhki** [comment]
Derek, I agree that you did very well on this project! Many of us came into the course with zero prior experience and it's awesome that we can now walk away with at least a base knowledge of these languages that were previously foreign to us.

**Comment by kassyoveson** [comment]
Derek, YOU ARE AMAZING! Your website is the best. I think it looks professional and is something you should be proud of. It must have taken a long time and I can only imagine the stress it may have caused. The end result is awesome. Great job! Enjoy the rest of your summer. You deserve it.
